frontend/sentry.client.config.ts

```diff
@@ -0,0 +1,21 @@
+// This file configures the initialization of Sentry on the browser.
+// The config you add here will be used whenever a page is visited.
+// https://docs.sentry.io/platforms/javascript/guides/nextjs/
+
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.SENTRY_DSN;
+
+Sentry.init({
+   dsn: SENTRY_DSN,
```

Comment:
What happens when this is blank? Does sentry fail? or just disable itself? or try to make a request to blank domain?

Reply:
Didn't see any console errors, and it didn't report to sentry in practice. I'll check out the network tab to see if it's trying to make requests

Reply:
I'm not seeing any requests made when SENTRY_DSN isn't set.

With SENTRY_DSN set:
![image](https://user-images.githubusercontent.com/52104630/172478015-4e765d0e-494e-43c8-a32f-cd85d22f066d.png)

Without the DSN env var:
![image](https://user-images.githubusercontent.com/52104630/172477862-63a09045-8102-4a8c-83bc-b93fab41dbcd.png)

Regardless, I'm happy to make a check if that seems more correct.

```suggestion
SENTRY_DSN && Sentry.init({
   dsn: SENTRY_DSN,
```

Reply:
Let's leave how you have it cause running it in dev gives us a little more peace of mind that this code won't cause issues in prod.